MEDO v0.5.owl

```diff
@@ -502,6 +575,52 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
     
 
 
+    <!-- http://purl.obolibrary.org/obo/MEDO_0000101 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MEDO_0000101">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl/MEDO_0000100"/>
+        <obo:IAO_0000115>A formal process of evaluating the degree to which a student or teacher has achieved one or more learning or skill objectives. This process coheres with standards-aligned methods of evaluation and returns an official valuation of performance.</obo:IAO_0000115>
+        <rdfs:label>formal assessment</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/MEDO_0000102 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MEDO_0000102">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl/MEDO_0000100"/>
+        <obo:IAO_0000115>An informal process of evaluating the degree to which a student or teacher has achieved one or more learning or skill objectives. This process typically is an ad hoc, low-stakes evaluation that does not return an officially recorded valuation of performance.</obo:IAO_0000115>
```

Comment:
@JaronJCheung Many of the above issues arise here. The definition suggests the parent is 'informal process' but placement in the hierarchy is under 'assay', suggesting multiple inheritance. 

Saying an assessment is 'low-stakes' is unclear without an appropriate comparison, i.e. low on what scale? 

You say 'typically is an ad hoc, low stakes...that does not return...' Are you suggesting informal assessments sometimes return officially recorded valuations? 

Are formal and informal assessments supposed to be disjoint classes?